system/Validation/FormatRules.php

```diff
@@ -29,6 +29,10 @@ class FormatRules
      */
     public function alpha($str = null): bool
     {
+        if (is_array($str)) {
+            return false;
+        }
+
```

Comment:
Why is this check only added here? Shouldn't it also be in other rules?

Reply:
Because there was an error when running phpunit.
But changing `alpha` is not good.

The Traditional Rules should not be changed as much as possible.
So I reverted it and changed the test code.

> Traditional Rules exist only for backward compatibility. Do not use them in new projects. Even if you are already using them, we recommend switching to Strict Rules.
https://codeigniter4.github.io/CodeIgniter4/libraries/validation.html#traditional-rules